Assets/Scripts/SS3D/Systems/Input/InputSystem.cs

```diff
@@ -0,0 +1,14 @@
+using SS3D.Core;
+using UnityEngine;
+
+public class InputSystem : SS3D.Core.Behaviours.System
```

Comment:
Add a summary explaining what this does.

Reply:
It's just contains one single Controls instance. Is summary really necessary?

Reply:
It's a bit verbose, I agree with you, but we might auto generate some doc in the future, and having a summary on each public classes even the trivial ones would be good for that.